lib/Sources/Tabs/TabCellModel.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 public struct TabModel: Equatable {
     public let index: Int
-    public let url: URL
+    public let url: URL?
```

Comment:
We should stick to non-nil here to avoid complications with the data model. instead we should use the view model (i.e. the `TabCell`) to represent view-related concepts.